src/styles/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  color: $color-text-card;
+
+  &:hover {
+    transform: scale(1.2);
+    transition-duration: 300ms;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    box-sizing: border-box;
+    margin-left: 3px;
+    transition: 3s;
+  }
+
+  &__title {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    padding-top: 40px;
+    height: 36px;
+  }
+
+  &:hover &__title {
+    color: #34568b;
+    cursor: pointer;
+    transition: 3s;
+  }
+
+  &__code {
+    justify-content: space-between;
+    margin: 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    padding-top: 4px;
+    color: $color-secondary;
+  }
+
+  &__rate {
+    padding-top: 17px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 16px;
+
+    &-reviews {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-top: 23px;
+
+    &-text {
+      margin: 0;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $color-text-card;
+    }
+
+    &-value {
+      margin: 0;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__btn {
+    width: $width;
+    height: 40px;
+    border: 0;
+    border-radius: 5px;
+    background-color: $btn-color;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    color: #fff;
+    margin-top: 16px;
+    cursor: pointer;
+    text-transform: uppercase;
```

Comment:
And here you are missing a transition property, so your transition when hover doesn't look abrupt